resources/definitions/renkforce_rf100.def.json

```diff
@@ -0,0 +1,231 @@
+{
+    "id": "RF100",
+    "version": 2,
+    "name": "renkforce RF100",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "author": "Simon Peter (based on RF100.ini by Conrad Electronic SE)",
+        "category": "Other",
+        "file_formats": "text/x-gcode",
+        "manufacturer": "renkforce",
+        "platform_offset": [0, 0, 0],
+        "visible": true
+    },
+
+    "overrides": {
+        "adhesion_type": {
+            "default_value": "skirt"
+        },
+        "bottom_thickness": {
+            "default_value": 0.5
+        },
+        "brim_line_count": {
+            "default_value": 20.0
```

Comment:
Not a blocking problem, but it's better to overwrite `brim_width` rather than `brim_line_count`, otherwise users can't modify `brim_width` any more (because `brim_line_count` overrides `brim_width`).